docs/source/common/debugging.rst

```diff
@@ -85,16 +92,18 @@ argument of :class:`~pytorch_lightning.trainer.trainer.Trainer`)
     # similar, but with a fixed 10 batches
     trainer = Trainer(overfit_batches=10)
 
-With this flag, the train, val, and test sets will all be the same train set. We will also replace the sampler
+When using this flag, validation will be disabled. We will also replace the sampler
 in the training set to turn off shuffle for you.
 
 ----------------
 
-Print a summary of your LightningModule
----------------------------------------
+Print a Summary of your LightningModule
+=======================================
+
 Whenever the ``.fit()`` function gets called, the Trainer will print the weights summary for the LightningModule.
+If is enabled by default and can be turned-off using ``Trainer(enable_model_summary=False)``.
```

Comment:
```suggestion
If is enabled by default and can be turned off using ``Trainer(enable_model_summary=False)``.
```

Reply:
I would mention this last, since from a debugging perspective you want to highlight the fact that you can set the depth. Mentioning how to disable it as the very first thing seems odd.
Also, would cause repetition of the default case in the next sentence